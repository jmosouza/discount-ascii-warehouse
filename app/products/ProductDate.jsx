import React from 'react';
import PropTypes from 'prop-types';

const ProductDate = ({ date }) => (
  <p>{date}</p>
);

ProductDate.propTypes = {
  date: PropTypes.string.isRequired,
};

export default ProductDate;
