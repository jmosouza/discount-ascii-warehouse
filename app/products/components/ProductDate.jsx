import React from 'react';
import PropTypes from 'prop-types';
import relativeDateInterval from '../../utils/relativeDateInterval';

const ProductDate = ({ date }) => (
  <p>{relativeDateInterval({ dateStr: date })}</p>
);

ProductDate.propTypes = {
  date: PropTypes.string.isRequired,
};

export default ProductDate;
