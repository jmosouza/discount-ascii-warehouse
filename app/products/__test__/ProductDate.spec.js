/* global test, expect */
import renderer from 'react-test-renderer';
import React from 'react';
import ProductDate from '../ProductDate';

test('ProductDate', () => {
  expect(renderer.create(
    <ProductDate
      date="Mon Oct 23 2017 11:23:20 GMT-0300 (-03)"
    />,
  )).toMatchSnapshot();
});
