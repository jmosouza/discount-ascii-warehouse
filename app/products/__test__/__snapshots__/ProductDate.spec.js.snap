// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductDate 1`] = `
<p>
  10/23/2017
</p>
`;
